Resources/Prototypes/Entities/Objects/Consumable/Food/produce.yml

```diff
@@ -167,6 +165,26 @@
       reagents:
       - ReagentId: JuiceBanana
         Quantity: 10
+  - type: SpawnItemsOnUse
+    items:
+    - id: FoodBananaPeeled
+    sound:
+      path: /Audio/Effects/unwrap.ogg
+
+- type: entity
+  name: banana
+  parent: FoodBanana
+  suffix: Peeled
+  id: FoodBananaPeeled
+  description: Rich in potassium.
+  components:
+  - type: Food
+    trash: TrashBananaPeel
+  - type: Sprite
+    sprite: Objects/Specific/Hydroponics/banana.rsi
+    state: peeled
+  - type: SpawnItemsOnUse
+    uses: 0
```

Comment:
Can't you just parent it the other way?

Reply:
Yes, but SpawnItemOnUse deserved to be fixed for 0 uses, too.

Reply:
yeah what partmedia said